test/commands/context.test.ts

```diff
@@ -89,4 +89,24 @@ describe('config', () => {
         done();
       });
   });
+  describe('context current', () => {
+    afterEach(() => {
+      testHelper.deleteDummyContextFile();
+    });
+
+    beforeEach(() => {
+      testHelper.createDummyContextFile();
+    });
+    test
+      .stderr()
+      .stdout()
+      .command(['config', 'context', 'current'])
+      .it('Shows the current context is home',(ctx,done) => {
+        expect(ctx.stdout).includes(
+          'home: ');
```

Comment:
I think we should check the whole message that is being printed out. This is the message that is being printed out so you can just check for that message `home: D:\\Programs\\AsyncAPI\\cli\\test\\specification.yml\n`

Reply:
Thanks @Souvikns . 
As `testHelper.context.current` may return undefined, replacing it with `home`